Docs/alert.html

```diff
@@ -0,0 +1,219 @@
+                    </ul>
+                </div>
+                <div class="sidebar-section">
+                    <h4 class="sidebar-heading">Components</h4>
+                    <ul class="sidebar-section-list">
+                        <li class="sidebar-list-item"><a href="/Docs/avatar.html">Avatar</a></li>
+                        <li class="sidebar-list-item item-active"><a href="/Docs/alert.html">Alert</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/badge.html">Badge</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/button.html">Button</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/card.html">Card</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/image.html">Image</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/input.html">Input</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/lists.html">Lists</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/modal.html">Modal</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/rating.html">Rating</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/grid.html">Grid</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/navigation.html">Navigation</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/snackbar.html">Snackbar</a></li>
+                        <li class="sidebar-list-item"><a href="/Docs/slider.html">Slider</a></li>
+                    </ul>
+                </div>
+            </aside>
+            <article class="flex-column" id="article">
+                <h2 class="component-title">Alert</h2>
+                <div class="component-description flex-column">
+                    <p class="para-m">
+                        Alerts are used to notify the user with information that might have originated
+                         from a user action for example, the result of an attempted picture upload or placing an order.
+                         This component library offers four variants of alerts.
+                    </p>
+                    <h3 class="component-title">Basic Alerts</h3>
+                    <p class="para-m">
+                        These alerts consist of basic texts. To use these alerts, one must use the below code and can change the class for 
+                        the type of alert <span class="highlight-text">"primary-alert","success-alert","warning-alert", "error-alert"</span>
+                             for primary, success, warning and error alerts respectively, as the naming suggests.
+                    </p>
+                    <div class="code-example flex-row">
+                        <pre><code class="language-html">&lt;div class="alert primary-alert"&gt; 
+&lt;p class="alert-text"&gt;A simple 
```

Comment:
Proper indentation would have been better

Reply:
Yeah, I tried that but the highlight js code example box takes up more space and isn't responsive. This is done on purpose